plugin/data-manipulation/data.manipulation.view.js

```diff
@@ -0,0 +1,234 @@
+import {Command} from '@grid/core/command';
+import {Action} from '@grid/core/action';
+import {AppError, Composite} from '@grid/core/infrastructure';
+import {isUndefined} from '@grid/core/utility';
+import * as columnService from '@grid/core/column/column.service';
+import {set as setValue} from '@grid/core/services/value';
+import {set as setLabel} from '@grid/core/services/label';
+import {PluginView} from '../plugin.view';
+
+export class DataManipulationView extends PluginView {
+	constructor(model) {
+		super();
+
+		this.model = model;
+
+		this.commitCommand = new Command({
```

Comment:
The main part of this code is from src/plugins/data-manipulation/data.manipulation.js file. Maybe it would be better to move src/plugins/data-manipulation/data.manipulation.js to this directory and name it data.manipulation.view.js to keep history of changes and in src/plugins/data-manipulation/ create new data.manipulation.js file

Reply:
Yes but a standard naming for such of services that we're using ends up with .view